dashboard/src/components/TestsTable/DefaultTestsColumns.tsx

```diff
@@ -0,0 +1,143 @@
+import type { ColumnDef } from '@tanstack/react-table';
+
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from '@radix-ui/react-tooltip';
```

Comment:
This should be imported from our own components because it's breaking the styling
```suggestion
import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/Tooltip';
```

Reply:
Nice!